.github/workflows/build.yml

```diff
@@ -30,10 +27,14 @@ jobs:
       run: git fetch origin +refs/tags/*:refs/tags/*
 
     - name: Set VERSION
-      run: echo ::set-env name=VERSION::$(git describe --dirty)
+      id: set_version
+      run: echo ::set-output name=VERSION::$(git describe --dirty)
 
     - name: Set GCE_VERSION
-      run: echo ::set-env name=GCE_VERSION::${VERSION//./-}
+      id: set_gce_version
+      env:
+        VERSION: ${{ steps.set_version.outputs.VERSION }}
+      run: echo ::set-output name=GCE_VERSION::${VERSION//./-}
```

Comment:
Out of curiousity, why using `set-output` and then adding the variable to env for each step one by one?

Reply:
Github deprecated `set-env` as being potential security issue https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/ ;) so this was the easiest way to replace it.